x/smart/README.md

```diff
@@ -82,7 +82,10 @@ tcp:
 
 A fallback configuration is used if none of the proxyless strategies are able to connect. For example it can specify a backup proxy server to attempt the user's connection. Using a fallback will be slower to start, since first the other DNS/TLS strategies must fail/timeout.
 
-The fallback strings should be a valid StreamDialer configs as defined in https://pkg.go.dev/github.com/Jigsaw-Code/outline-sdk/x/configurl#hdr-Proxy_Protocols
+The fallback strings should be:
+
+*   A valid StreamDialer config string as defined in [configurl](https://pkg.go.dev/github.com/Jigsaw-Code/outline-sdk/x/configurl#hdr-Proxy_Protocols)
+*   A valid Psiphon configuration.
```

Comment:
This needs to be updates. It's a struct with a psiphon field with a Psiphon configuration value.

Reply:
done